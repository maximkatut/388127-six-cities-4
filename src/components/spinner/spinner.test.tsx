import React from 'react';
import renderer from 'react-test-renderer';

import Spinner from './spinner';

describe(`Spinner`, () => {

  it(`Spinner component should render correctly`, () => {

    const component = renderer.create(
        <Spinner/>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
