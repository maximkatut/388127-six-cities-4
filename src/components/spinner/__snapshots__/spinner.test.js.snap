// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Spinner component should render correctly 1`] = `
<div
  className="sk-chase"
>
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
</div>
`;
